### DISCUSSION TITLE
Loft

### DISCUSSION THREAD

**Discussion by jgomez720** [KittyCAD/modeling-app#613] [New Feature Requests]
### Description
Enables the uesr to create a loft, an extrusion between two distinct profiles. Example: Extruding a square to a circle

### Functionality
1. User selects _Loft_
2. User selects the two profiles. Typically, the planes they lay on have to be parallel to each other.
3. Confirm.

### Supporting Docs/Images
<img width="660" alt="Screenshot 2023-09-18 at 5 34 08 PM" src="https://github.com/KittyCAD/modeling-app/assets/114548659/24931708-482f-4606-89c6-cff573f2224e">
<img width="849" alt="Screenshot 2023-09-18 at 5 35 30 PM" src="https://github.com/KittyCAD/modeling-app/assets/114548659/db507e99-efd1-4a1b-95ab-a783becd1f25">
<img width="538" alt="Screenshot 2023-09-18 at 5 35 44 PM" src="https://github.com/KittyCAD/modeling-app/assets/114548659/7fca1798-4981-4f85-aa97-7be573e180a5">

**Comment by Irev-Dev** [comment]
![image](https://github.com/KittyCAD/modeling-app/assets/29681384/5185ae66-5f65-46a9-9126-2e32f7c9f654)

**Comment by jessfraz** [comment]
implemented in https://github.com/KittyCAD/modeling-app/pull/3681
